### PR TITLE
[DOC] Add doc and helm chart to describe how to deploy on k8s

### LINKED ISSUE #302: [DOC] Add doc describing how to deploy on k8s
### Describe alternatives you've considered

/cc @who-need-to-know

/assign @who-can-help-you

### PR DESCRIPTION
**Pull request by fanyilun**
# [DOC] Add doc and helm chart to describe how to deploy on k8s

### What changes were proposed in this pull request?
closes #302 

### Why are the changes needed?

help user to deploy rss on k8s

### What are the items that need reviewer attention?

docker file and helm charts

### Related issues.

#302 

### How was this patch tested?

/cc @related-reviewer

/assign @waitinfuture @FMX

### CODE REVIEW

File: docker/helm/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  log4j.properties: |-
+    # Set everything to be logged to the console
+    log4j.rootCategory=INFO, console
+    log4j.appender.console=org.apache.log4j.ConsoleAppender
+    log4j.appender.console.target=System.err
+    log4j.appender.console.layout=org.apache.log4j.PatternLayout
+    log4j.appender.console.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{1}: %m%n
+
+  metrics.properties: >-
+    *.sink.prometheusServlet.class=com.aliyun.emr.rss.common.metrics.sink.PrometheusServlet
```
Comment (waitinfuture) [inline docker/helm/templates/configmap.yaml:35]: Add newline at end of file

File: docker/helm/templates/prometheus-podmonitor.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      path: /metrics/prometheus
+  jobLabel: shuffleservice-worker-podmonitor
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: shuffleserviceworker
+{{ end }}
+{{- end }}
```
Comment (waitinfuture) [inline docker/helm/templates/prometheus-podmonitor.yaml:39]: newline

File: docker/helm/values.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# Default values for remote-shuffle-service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# we provide a default rss image, you can also replace to your image
+image:
+  repository: registry.cn-shanghai.aliyuncs.com/emr-ack/remote-shuffle-service
```
Comment (FMX) [inline docker/helm/values.yaml:7]: I think the public should not access this repo.

File: docker/helm/templates/configmap.yaml (hunk 1 above)
Comment (fanyilun) [reply docker/helm/templates/configmap.yaml]: done

File: docker/helm/templates/prometheus-podmonitor.yaml (hunk 2 above)
Comment (fanyilun) [reply docker/helm/templates/prometheus-podmonitor.yaml]: done

File: docker/helm/values.yaml (hunk 3 above)
Comment (fanyilun) [reply docker/helm/values.yaml]: changed to docker hub

**Review by waitinfuture — APPROVED** [review]
LGMT, thanks!

**Pull request merged**